src/api/routes/articleRouter.js

```diff
@@ -32,4 +33,8 @@ articleRouter.post('/',
   checkValidToken,
   uploadImage.single('coverImage'),
   errorHandler(articleController.create));
+
+articleRouter.post('/:articleId/highlight', checkValidToken, validateInputs(true, 'highlight', highlightFields),
```

Comment:
@alainburindi I've actually modified this middleware. I received feedback on it from @MemunaHaruna.
try to check it out on my `get article ratings` branch